HW_06/ArrayValueCalculator.java

```
@@ -0,0 +1,30 @@
+public class ArrayValueCalculator {
+    static int doCalc(String[][] arr) {
+        try {
+            if (arr.length != 4 || !checkArray(arr)) {
+                throw new ArraySizeException();
+            } else {
+                System.out.println("Array size is ok.");
+            }
+        } catch (ArraySizeException ignored){}
```

Please never ignore exceptions!